### ISSUE TITLE
Handle Control-C in shim wrapper

### ISSUE THREAD

**Issue by dimdimych** [lukesampson/scoop#1896]
When user hits Ctrl-C in cmd.exe session when running a program wrapped by Shim.exe, terminal session gets corrupted (hard to describe, command prompt is displayed but no commands can be started, Far Manager stops to respond, etc). The process started by Shim is not terminated

**Comment by kroppt** [comment]
Is this still a problem? I tried replicating using `tail -f` on a file, but it closed like it was supposed to.
I opened up `cmd.exe`, navigated to shim directory, ran `.\tail.exe -f C:\Users\User\test.txt`, and pressed Ctrl+C. The process (both) was killed.

**Comment by rasa** [comment]
This issue should be fixed in #3998, which was merged into master. To use, type:
```
scoop config shim kiennq
scoop reset *
```
or
```
scoop config shim 71
scoop reset *
```
Eventually, one of those shims (probably kiennq) we be made the default. 

If this issue is still a problem after trying both solutions above, please reopen this issue. Thanks!
